README.md

```diff
@@ -366,6 +366,32 @@ const connector = new Connector({
 });
 ```
 
+This can be useful when configuring credentials that differ from
+Application Default Credentials.
+
+The below example showcases how to initialize a `Connector` from a service account key that is
+[loaded into an environment variable][google-auth-load-creds].
+
+```js
+import {Connector} from '@google-cloud/cloud-sql-connector';
+import {GoogleAuth} from 'google-auth-library';
+
+// load the environment variable with our keys
+const keysEnvVar = process.env.CREDS
+if (!keysEnvVar) {
+  throw new Error('The $CREDS environment variable was not found!');
+}
+const keys = JSON.parse(keysEnvVar);
+
+const auth = new GoogleAuth({
+    scopes: ['https://www.googleapis.com/auth/sqlservice.admin']
+});
+
+const connector = new Connector({
+    auth: auth.fromJSON(keys),
+});
```

Comment:
The following also seems to work...
```js
const auth = new GoogleAuth({
    credentials: keys,
    scopes: ['https://www.googleapis.com/auth/sqlservice.admin']
});

const connector = new Connector({
    auth: auth,
});
```

@ruyadorno do you have any preference? One is passing in a `GoogleAuth` type and the other is `AuthClient`. Would users be more likely to pass in a `GoogleAuth` or `AuthClient` type? I don't really think it matters.

Reply:
the alternative in this comment looks better imo!

Reply:
updated